packages/expo-core/android/src/main/java/expo/core/interfaces/ActivityResultListener.java

```diff
@@ -0,0 +1,7 @@
+package expo.core.interfaces;
+
+import android.content.Intent;
+
+public interface ActivityResultListener {
```

Comment:
I think we already have `ActivityEventListener` 😄 And it doesn't look like `ActivityResultListener` is used somewhere 🤔